test/lib/runtime/broker_test.exs

```diff
@@ -587,8 +590,13 @@ defmodule Ask.Runtime.BrokerTest do
 
       survey1 = Repo.get(Ask.Survey, survey1.id)
       survey2 = Repo.get(Ask.Survey, survey2.id)
+
       assert Ask.Survey.completed?(survey1)
       assert survey2.state == "running"
+
+      # Only polled surveys have first_window_started_at
+      assert survey1.first_window_started_at == now
+      refute survey2.first_window_started_at
```

Comment:
I believe the `first_window_started_at` logic deserves its own tests (like you did un `survey_test` before) 🙃 

We need to test that:
 - this field is set when a survey is polled for the first time
 - when a survey is polled for second time, the field is not updated
 - when a survey is created has this field in `nil`